consensus/consensus_fbft.go

```diff
@@ -1,9 +1,6 @@
 package consensus
 
-import "sync"
-
 type LockedFBFTPhase struct {
-	mu    sync.Mutex
 	phase FBFTPhase
 }
```

Comment:
if there is no lock here, then how this struct helps? Also, The name like **LockedFBFTPhase** doesn't match with what it is doing. Basically we don't need this file and could be deleted and LockedFBFTPhase could be replaced with a simple **FBFTPhase** variable everywhere in codes.

Reply:
Removed